.circleci/ds005_outputs.txt

```diff
@@ -24,16 +24,20 @@ smriprep/sub-01/anat/sub-01_hemi-L_curv.shape.gii
 smriprep/sub-01/anat/sub-01_hemi-L_inflated.surf.gii
 smriprep/sub-01/anat/sub-01_hemi-L_midthickness.surf.gii
 smriprep/sub-01/anat/sub-01_hemi-L_pial.surf.gii
-smriprep/sub-01/anat/sub-01_hemi-L_smoothwm.surf.gii
+smriprep/sub-01/anat/sub-01_hemi-L_space-fsLR_sphere.surf.gii
+smriprep/sub-01/anat/sub-01_hemi-L_sphere.surf.gii
```

Comment:
Are these sensible names?

Reply:
Maybe approaching too verbose, but adding `desc-reg` seems useful

```suggestion
smriprep/sub-01/anat/sub-01_hemi-L_space-fsLR_desc-reg_sphere.surf.gii
smriprep/sub-01/anat/sub-01_hemi-L_space-fsaverage_desc-reg_sphere.surf.gii
```

Reply:
Sounds good. Let's do that.